ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -663,13 +693,242 @@ objects:
+
+          # Add internal DNS entries
+          if [ -f transaction.yaml ]; then rm transaction.yaml; fi
+          gcloud dns record-sets transaction start --zone ${INFRA_ID}-private-zone
+          gcloud dns record-sets transaction add ${CLUSTER_IP} --name api.${CLUSTER_NAME}.${GCP_BASE_DOMAIN}. --ttl 60 --type A --zone ${INFRA_ID}-private-zone
+          gcloud dns record-sets transaction add ${CLUSTER_IP} --name api-int.${CLUSTER_NAME}.${GCP_BASE_DOMAIN}. --ttl 60 --type A --zone ${INFRA_ID}-private-zone
+          gcloud dns record-sets transaction execute --zone ${INFRA_ID}-private-zone
+
+          # Deploy security
+          export MASTER_NAT_IP=`gcloud compute addresses describe ${INFRA_ID}-master-nat-ip --region ${REGION} --format json | jq -r .address`
+          export WORKER_NAT_IP=`gcloud compute addresses describe ${INFRA_ID}-worker-nat-ip --region ${REGION} --format json | jq -r .address`
+
+          cat <<EOF >/tmp/gcp/03_security.yaml
+          imports:
+          - path: 03_security.py
+
+          resources:
+          - name: cluster-security
+            type: 03_security.py
+            properties:
+              infra_id: '${INFRA_ID}'
+              region: '${REGION}'
+              cluster_network: '${CLUSTER_NETWORK}'
+              network_cidr: '${NETWORK_CIDR}'
+              master_nat_ip: '${MASTER_NAT_IP}'
+              worker_nat_ip: '${WORKER_NAT_IP}'
+          EOF
+
+          gcloud deployment-manager deployments create ${CLUSTER_NAME}-security --config /tmp/gcp/03_security.yaml
+
+          export MASTER_SA=${INFRA_ID}-m@${PROJECT_NAME}.iam.gserviceaccount.com
+          gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${MASTER_SA}" --role "roles/compute.instanceAdmin"
+          gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${MASTER_SA}" --role "roles/compute.networkAdmin"
+          gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${MASTER_SA}" --role "roles/compute.securityAdmin"
+          gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${MASTER_SA}" --role "roles/iam.serviceAccountUser"
+          gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${MASTER_SA}" --role "roles/storage.admin"
+
+          export WORKER_SA=${INFRA_ID}-w@${PROJECT_NAME}.iam.gserviceaccount.com
+          gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${WORKER_SA}" --role "roles/compute.viewer"
+          gcloud projects add-iam-policy-binding ${PROJECT_NAME} --member "serviceAccount:${WORKER_SA}" --role "roles/storage.admin"
+
+          gcloud iam service-accounts keys create /tmp/gcp/service-account-key.json --iam-account=${MASTER_SA}
+
+          # Deploy bootstrap
+          export IMAGE_SOURCE=`cat /var/lib/openshift-install/data/data/rhcos.json | jq -r .gcp.url`
```

Comment:
Looks like this should be `/var/lib/openshift-install/rhcos.json` based on [this precedent][1], although I'm not clear on how that works.

[1]: https://github.com/openshift/release/blob/cfbe8159573423e5d2b372430108cdb531f09d69/ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml#L473

Reply:
Thanks! Fixed.